packages/aptos-wallet-adapter/src/WalletAdapters/SpikaWallet.ts

```diff
@@ -51,7 +51,7 @@ export class SpikaWalletAdapter extends BaseWalletAdapter {
 
   url = 'https://chrome.google.com/webstore/detail/spika/fadkojdgchhfkdkklllhcphknohbmjmb';
 
-  icon = 'https://spika.app/assets/logo_400_nb.png';
+  icon = 'https://raw.githubusercontent.com/hippospace/aptos-wallet-adapter/main/logos/spika.svg';
```

Comment:
https://raw.githubusercontent.com/hippospace/aptos-wallet-adapter/main/logos/spika.svg returns 404 not found ? @xorgal

Reply:
@tom-hippo 
I guess we need to merge PR first as logo is not pushed to the main branch until then?